README.md

```diff
@@ -22,3 +22,106 @@ add script "dev": "next" in the package.json
 yarn install
 yarn run dev
 ```
+
+## Integration tests
+
+This code uses [Watai](https://github.com/MattiSG/Watai/wiki) framework for integration tests.
+
+### Install integration tests environment
+
+To run the tests with Watai, we need Java JRE and a Selenium standalone server.
+
+#### Install Java JRE
+
+Check if a JRE is already installed on your environment with:
```

Comment:
What is a JRE ? Why not `Check if java is installed in your environment` ?

Reply:
A JRE is lighter as it doesn't contain development tools.